### PR TITLE
[RHCLOUD-24697] Run Flyway on Ephemeral for engine

### PR DESCRIPTION
**Pull request by g-duval**

### CODE REVIEW

File: engine/src/main/java/com/redhat/cloud/notifications/ephemeral/EphemeralDataInitializer.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import org.flywaydb.core.Flyway;
+import javax.annotation.Priority;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+
+import static javax.enterprise.inject.spi.ObserverMethod.DEFAULT_PRIORITY;
+
+@ApplicationScoped
+public class EphemeralDataInitializer {
```
[the remark this answers is not in the document]
Comment (g-duval) [reply engine/src/main/java/com/redhat/cloud/notifications/ephemeral/EphemeralDataInitializer.java:16]: Ok

**Review by gwenneg — APPROVED** [review]
LGTM

**Comment by gwenneg** [comment]
Maybe you already did it but it would be good to check the `notifications-engine` logs from ephemeral and confirm the migration works before merging this PR.

**Pull request merged**